linux/runtime.go

```diff
@@ -204,6 +207,7 @@ func (r *Runtime) forward(events shim.Shim_EventsClient) {
 			Pid:        e.Pid,
 			ID:         e.ID,
 			ExitStatus: e.ExitStatus,
+			ExitedAt:   e.ExitedAt,
```

Comment:
It is generated in the shim.  The line you just commented on is taking the ExitedAt value from the shim response and propagating it up to the execution GRPC service from the shim.